client-web/src/functional-e2e/templates/template-types/usage/contributions/callout-template.use.contributions.ts

```
@@ -0,0 +1,136 @@
+      //!! WARNING, THIS IS A BUG, UNCOMMENT WHEN FIXED
+      // await expect(
+      //   collectionSettingsDialog.getByText(templateData.responseOptions.defaultDescription, { exact: true })
+      // ).toBeVisible();
+      break;
+    }
+    case 'whiteboards': {
+      // Verify the Title:
+      await expect(
+        await collectionSettingsDialog.getByRole('textbox', { name: 'Title' }).inputValue()
+      ).toBe(templateData.responseOptions.defaultTitle);
+    }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)